src/main/java/de/tum/in/www1/artemis/web/rest/CompetencyResource.java

```diff
@@ -643,4 +654,60 @@ private void checkAuthorizationForCompetency(Role role, @NotNull Course course,
         }
         authorizationCheckService.checkHasAtLeastRoleInCourseElseThrow(role, course, null);
     }
+
+    /**
+     * PUT courses/:courseId/competencies/:competencyId/jol/:jolValue : Sets the judgement of learning for a competency
+     *
+     * @param courseId     the id of the course for which the competency belongs
+     * @param competencyId the id of the competency for which to set the judgement of learning
+     * @param jolValue     the value of the judgement of learning
+     * @return the ResponseEntity with status 200 (OK)
+     */
+    @PutMapping("courses/{courseId}/competencies/{competencyId}/jol/{jolValue}")
+    @EnforceAtLeastStudentInCourse
+    public ResponseEntity<Void> setJudgementOfLearning(@PathVariable long courseId, @PathVariable long competencyId, @PathVariable int jolValue) {
+        log.info("REST request to set judgement of learning for competency: {}", competencyId);
```

Comment:
Do you actually want to have this log on an "INFO" level? This seems to be more suitable for debug imo.